README.md

```diff
@@ -164,7 +153,45 @@ For more parameters, please see [Monasca Agent Documentation](https://github.com
 
 This will create and run a new service file `/etc/systemd/system/monasca-agent.service` with the configuration set as per the arguments mentioned above. 
 
-### Log agent
+### Uninstalling Metrics agent
+Stop the services: 
+```
+systemctl stop monasca-agent
+```
+Delete created files: 
+
+*(Note: there may be files in /etc/monasca/agent/ that are related to different agents, so only remove the ones listed.)*
```

Comment:
Isn't it good to remove `/etc/monasca`?

Reply:
I changed it to removing `/etc/monasca/agent/`, we don't know if user have any other stuff from monasca installed there.